### PR TITLE
[um-style-changes] some style fixes.

### PR DESCRIPTION
**Pull request by frknbasaran**

### CODE REVIEW

File: frontend/express/public/localization/dashboard/dashboard.properties [hunk 1]
```diff
@@ -17,17 +17,11 @@ common.session-duration = Session Duration
 common.table.no-data = No data
 common.search.no-match-found = No match found
 common.table.total-users = Total Users
-common.table.total-users-desc = Number of times your application is opened by a new or returning user from a unique device, in the selected time period.
```
Comment (Cookiezaurs) [inline frontend/express/public/localization/dashboard/dashboard.properties:20]: Please don't delete/overwrite localization strings !!!!

Comment (Cookiezaurs) [inline frontend/express/public/localization/dashboard/dashboard.properties:20]: Please don't delete/overwrite localization strings !!!!

**Review by Cookiezaurs — CHANGES_REQUESTED** [review]

File: frontend/express/public/localization/dashboard/dashboard.properties [hunk 2]
```diff
@@ -219,10 +213,6 @@ common.confirm = Confirm
 common.remove = Remove
 common.next = Next
 common.previous = Previous
-common.select-at-least = At least {0} items should be selected. 
-common.select-at-most = At most {0} items can be selected.
-common.and = and
```
Comment (can-keklik) [inline frontend/express/public/localization/dashboard/dashboard.properties:224]: Hmm, I agree with Anna. It is weird though... I don't think that you explicitly removed these fields, so why are they deleted?

File: frontend/express/public/localization/dashboard/dashboard.properties (hunk 1 above)
Comment (frknbasaran) [reply frontend/express/public/localization/dashboard/dashboard.properties:20]: okay, please be calm.

File: frontend/express/public/localization/dashboard/dashboard.properties (hunk 2 above)
Comment (frknbasaran) [reply frontend/express/public/localization/dashboard/dashboard.properties:224]: not sure, solving rn.

**Pull request closed without merging**